From 53a9547665aec39a09c66dbd44ee8005ec4141b2 Mon Sep 17 00:00:00 2001
From: Masaharu Hayakawa <masaharu.hayakawa.ry@renesas.com>
Date: Fri, 11 May 2018 15:34:02 +0900
Subject: [PATCH 221/909] mmc: tmio: Delete wait in tuning process

commit e6801ade7db2cfb9315298b5610bf1f5074fe49a from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

There is no description that wait is necessary
in the tuning process of H/W manual.
Delete wait to shorten the initialization time.

Signed-off-by: Masaharu Hayakawa <masaharu.hayakawa.ry@renesas.com>
Signed-off-by: Takeshi Saito <takeshi.saito.xv@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mmc/host/tmio_mmc_core.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/mmc/host/tmio_mmc_core.c b/drivers/mmc/host/tmio_mmc_core.c
index 181bd7adb58d..39982acad1f3 100644
--- a/drivers/mmc/host/tmio_mmc_core.c
+++ b/drivers/mmc/host/tmio_mmc_core.c
@@ -943,8 +943,6 @@ static int tmio_mmc_execute_tuning(struct mmc_host *mmc, u32 opcode)
 
 		if (host->compare_scc_data && !host->compare_scc_data(host))
 			set_bit(i, host->smpcmp);
-
-		usleep_range(1000, 1200);
 	}
 
 	ret = host->select_tuning(host);
-- 
2.17.1

